fe/fe-core/src/main/java/com/starrocks/qe/DDLStmtExecutor.java

```diff
@@ -781,6 +781,7 @@ public ShowResultSet visitCreateAnalyzeJobStatement(CreateAnalyzeJobStmt stmt, C
                     statsConnectCtx.setThreadLocalInfo();
                     StatisticExecutor statisticExecutor = new StatisticExecutor();
                     analyzeJob.run(statsConnectCtx, statisticExecutor);
+                    statsConnectCtx.close();
```

Comment:
statsConnectCtx.close() would not be invoked if exception throwed  at ln781~783

Reply:
fixed